src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -2472,6 +2473,38 @@ public void HandleIDebugMessageEvent2(IDebugEngine2 pEngine, IDebugProcess2 pPro
             }
         }
 
+        public void HandleIDebugProcessInfoUpdatedEvent158(IDebugEngine2 pEngine, IDebugProcess2 pProcess, IDebugProgram2 pProgram, IDebugThread2 pThread, IDebugEvent2 pEvent)
+        {
+            IDebugProcessInfoUpdatedEvent158 debugProcessInfoUpdated = pEvent as IDebugProcessInfoUpdatedEvent158;
+
+            if (debugProcessInfoUpdated != null && 
+                debugProcessInfoUpdated.GetUpdatedProcessInfo(out string name, out uint systemProcessId) == HRConstants.S_OK)
+            {
+                // Send ProcessEvent to Client
+                ProcessEvent processEvent = new ProcessEvent();
+                processEvent.Name = name;
+                processEvent.SystemProcessId = (int)systemProcessId;
```

Comment:
@pieandcakes Should we save `systemProcessId` in AD7DebugSession? This is the actual debuggee process reported from GDB.

Reply:
I'll do this in another PR, this can help Tracepoints evaluate `$PROCESSID`